### DISCUSSION TITLE
kubecm list shortened usage

### DISCUSSION THREAD

**Discussion by MortalSW** [sunny0826/kubecm#562] [General]
Hello.

You are provide 
`kubecm s` as `kubecm switch`
and
`kubecm l` as `kubecm list`

Why you aren't probide `kubecm n` as `kubecm namespace`? I wrote function for me, but i want this will be native shortened alias.

**Comment by sunny0826** [comment]
I have provide `kubecm ns`

**Reply by sunny0826** [reply]
I think `ns` is a better way to express what namespace means.

**Reply by sunny0826** [reply]
It is also demonstrated in the documentation.

https://kubecm.cloud/en-us/cli/kubecm_namespace
